File: src/main/java/com/jyc/videoscala/java/Movie.java
package com.jyc.videoscala.java;

public class Movie {
    public static final int REGULAR = 1;
    public static final int NEW_RELEASE = 2;
    public static final int CHILDRENS = 3;

    private String title;
    private Integer priceCode;

    public Movie(String title, Integer priceCode) {
        this.title = title;
        this.priceCode = priceCode;
    }

    public String getTitle() {
        return title;
    }

    public Integer getPriceCode() {
        return priceCode;
    }
}

File: src/main/scala/com/jyc/videoscala/scala/Movie.scala
package com.jyc.videoscala.scala

class Movie {
  private var title: String = null
  private var priceCode: Int = 0

  def this(title: String, priceCode: Int) {
    this()
    this.title = title
    this.priceCode = priceCode
  }

  def getTitle: String = {
    title
  }

  def getPriceCode: Int = {
    priceCode
  }
}

object Movie {
  val REGULAR: Int = 1
  val NEW_RELEASE: Int = 2
  val CHILDRENS: Int = 3
}
